### PR TITLE
Portfolio: add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by jecode93**
## For this milestone I :
- Add a contact form to my portfolio and give my coding partner access to contribute.
- In order to lay out the elements on the page I use Flexbox.
- Implement the button interactions

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -173,6 +173,25 @@ <h2>Skills</h2>
...
+          <div class="contact-head">
+            <h1>Contact me</h1>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project
+              that needs coding. I'd love to help with it!</p>
+          </div>
+          <div class="form">
+            <form action="https://formspree.io/f/xgebjlkk" method="post">
+              <p><input type="text" name="fullname" id="fullname" placeholder="Lorem Ipsum" required></p>
+              <p><input type="email" name="email" id="email" placeholder="loremipsum@gmail.com" required></p>
+              <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message"
```
Comment (roshan-bajgain) [inline html/index.html]: - [ ] According to the project requirement there should be HTML validation maxlength="30" for name input and Message input has `maxlength="500"` attribute So any user cannot use name and message more than the validation.
- [ ] [Optional] Also I would suggest you please remove the `P` tag from input field for better semantic HTML.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @jecode93 and @mj-e-boc, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] If you decide not to deploy the app then please remove the live-demo from README file it makes you README more professional.
- [ ] Also I would suggest please add second author on you README file so it look more professional and descriptive.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: html/index.html (hunk 1 above)
Comment (jecode93) [reply html/index.html]: Hello @roshan-bajgain 

Thanks for your review! I will do the changes...

**Review by Gambit142 — APPROVED** [review]
Hi @jecode93 @mj-e-boc 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- README file is professional ✔️ 
- Use of HTML validations ✔️ 

### Optional Suggestion

- [ ] In your MIT LICENSE, it would be nice to personalize the template used by replacing the placeholder phrase `[YOUR NAME]` with your actual name and all placeholder phrases [APP TYPE] with _**WEBSITE**_. 👍 😎 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**